examples/colormaterial_color.rs

```diff
@@ -5,21 +5,18 @@ use bevy::{
 use bevy_tweening::{lens::*, *};
 use std::time::Duration;
 
-fn main() -> Result<(), Box<dyn std::error::Error>> {
+fn main() {
     App::default()
         .insert_resource(WindowDescriptor {
             title: "ColorMaterialColorLens".to_string(),
             width: 1200.,
             height: 600.,
-            present_mode: bevy::window::PresentMode::Fifo, // vsync
-            ..Default::default()
+            ..default()

```

Comment:
Why remove vsync? It is indeed the default, but it's not very clear (I had to dig through the Bevy sources) and is not documented as such.

Reply:
I thought it was repetitive since it's the default, but I don't care either way. Added back.